### PR TITLE
Add test for $m and $v

### PR DESCRIPTION
**Pull request by SchumBlubBlub**
Tests for https://github.com/radare/radare2/pull/6690

### CODE REVIEW

File: t/cmd_special_vars [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+CMDS='wa mov eax, dword [0x00001234]; ?v ?m;'
+EXPECT="0x1234
+"
+run_test
+
+NAME='inmem $v'
+FILE=-
+BROKEN=
+ARGS=
+CMDS='wa mov eax, 0x1234; ?v ?v;'
```
Comment (radare) [inline t/cmd_special_vars]: ?v ?v makes no sense

File: t/cmd_special_vars [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+[ -e tests.sh ] && . ./tests.sh || . ../tests.sh
+
+NAME='inmem $m'
+FILE=-
+BROKEN=
+ARGS=
+CMDS='wa mov eax, dword [0x00001234]; ?v ?m;'
```
Comment (radare) [inline t/cmd_special_vars]: ?v ?m makes no sense

Comment (radare) [inline t/cmd_special_vars]: variables are prefixed with $, not ?

Comment (SchumBlubBlub) [inline t/cmd_special_vars]: Edit error fom shell->test. Fixed. One is working, so just for future. `$v` is broken I think.

**Comment by SchumBlubBlub** [comment]
fixed tests, `$v` still broken.

**Comment by radare** [comment]
its not broken, its just not working for that specific case, fix the analysis. compare with the output of `ao`

**Comment by SchumBlubBlub** [comment]
> compare with the output of `ao`

I don't know which field an immediate is supposed to end up in, but `ab 8b0534120000` puts the immediate in the ptr field, which is exactly why $v doesn't work. Isn't that wrong?

**Comment by SchumBlubBlub** [comment]
Found the broken `$m` case, added more tests.

**Comment by Maijin** [comment]
No need for another file for tests, use https://github.com/SchumBlubBlub/radare2-regressions/blob/master/t/feat_variables

**Comment by SchumBlubBlub** [comment]
Kept the debug cases under `t.archos/Linux-x86_64/dbg_feat_variables` but merged the others into existing file.

**Comment by Maijin** [comment]
ok perfect (y)

**Comment by SchumBlubBlub** [comment]
-ish

**Pull request closed without merging**